src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,104 @@
-import React from 'react';
+import React, { useState, Dispatch, SetStateAction } from 'react';
+import classNames from 'classnames';
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+import { Post } from '../types/Post';
+
+interface T {
+  selectedPost: Post;
+  setIsLoading: Dispatch<SetStateAction<boolean>>;
+  isLoading: boolean;
+  setUserComments: Dispatch<SetStateAction<Comment[]>>;
+}
+
+enum Inputs {
+  Name = 'name',
+  Email = 'email',
+  Text = 'text',
+}
+
+export const NewCommentForm: React.FC<T> = ({
+  selectedPost,
+  setIsLoading,
+  isLoading,
+  setUserComments,
+}) => {
+  const [inputName, setInputName] = useState('');
+  const [inputEmail, setInputEmail] = useState('');
+  const [inputText, setInputText] = useState('');
+  const [isInputNameSent, setIsInputNameSent] = useState(false);
+  const [isInputEmailSent, setIsInputEmailSent] = useState(false);
+  const [isInputTextSent, setIsInputTextSent] = useState(false);
+
+  const onSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    setIsLoading(true);
+    setIsInputNameSent(true);
+    setIsInputEmailSent(true);
+    setIsInputTextSent(true);
+
+    try {
+      const newComment: Comment = {
+        postId: selectedPost.id,
+        name: inputName,
+        email: inputEmail,
+        body: inputText,
+      };
+
+      if (
+        inputName.trim()
+        && inputEmail.trim()
+        && inputText.trim()
+      ) {
+        const response: any = await client.post('/comments', newComment);
```

Comment:
Try to fix linter error
![image](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/56752562/4528dab6-bcb3-4252-8f3d-252681d564bd)